src/components/HtxTextBox/HtxTextBox.js

```diff
@@ -104,9 +124,9 @@ export class HtxTextBox extends React.Component {
       autoFocus: true,
       ref: this.inputRef,
       value,
-      onBlur: isFF(FF_DEV_1566) ? ()=>{
-        onChange(this.state.value);
-      } : this.save,
+      onBlur: () => {
```

Comment:
What happens when we remove `this.save`? Does that lock the text into the textbox?

Reply:
Yeah, you are right, `this.save` locks the text and goes back to the uneditable view

Reply:
cool, add comment!!